docs/source/walkthrough.md

```diff
@@ -47,8 +47,8 @@ Once we’ve deployed the Cape function successfully, we’ll have a function ID
 ```python
 cape = Cape()
 function_id = "4akLQwrqydyXYdyqn9qpSK"
-checksum = "8d3559c4d22df470be639aedbbb32c0857d3aca45c78e98be24c8a31fe051f75"
-function_ref = FunctionRef(function_id, checksum)
+func_checksum = "8d3559c4d22df470be639aedbbb32c0857d3aca45c78e98be24c8a31fe051f75"
```

Comment:
we should probably rename this to `function_checksum` everywhere user-facing, the discrepancy w/ `function_id` looks weird/unpolished to me